eth2near/eth2near-block-relay-rs/src/beacon_rpc_client.rs

```diff
@@ -308,7 +305,11 @@ impl BeaconRPCClient {
         let json_str = self.get_json_from_raw_request(&url_request)?;
 
         let v: Value = serde_json::from_str(&json_str)?;
-        Ok(v["data"]["is_syncing"].as_bool().unwrap())
+        return if let Some(is_sync) = v["data"]["is_syncing"].as_bool() {
+            Ok(is_sync)
+        } else {
+            Err(Box::new(ErrorOnJsonParse))?
+        }
```

Comment:
`return` statement is redundant, but maybe it's still worth to rewrite it to something like this:

```suggestion
        v["data"]["is_syncing"].as_bool().ok_or(Err(Box::new(ErrorOnJsonParse))?)
```